docs/instrument.md

```diff
@@ -0,0 +1,41 @@
+# Producing instrumented source
+
+The `nyc instrument` command can produce instrumented source files.
+These files are suitable for client side deployment during end to end testing.
+You can either pre-instrument your source, or write instrumented source to a stream.
+
+Run `nyc instrument --help` to display a full list of available command options.
+ 
+## Pre-instrumented source 
+ 
+You can create pre-instrumented source code by running:
+
+```bash
+nyc instrument <input> [output]
+```
+
+`<input>` can be any file or directory within the project root directory.
+The `[output]` directory is optional and can be located anywhere, if not set the instrumented code will be sent to `stdout`.
+For example, `nyc instrument . ./output` will produce instrumented versions of any source files it finds in `.` and store them in `./output`.
+
+The `--delete` option will remove the existing output directory before instrumenting.
+
+The `--complete-copy` option will copy all remaining files from the `input` directory to the `output` directory.
+When using `--complete-copy` nyc will not copy the contents of a `.git` folder to the output directory.
+
+**Note:** `--complete-copy` will dereference any symlinks during the copy process, this may stop scripts running properly from the output directory.
+
+## Streaming instrumentation
+
+This form of the command will stream instrumented source directly to `stdout`, that can then be piped to another process.
```

Comment:
This sentence should just start with `nyc instrument` will stream...

Reply:
@AndrewFinlay do you have a moment to address this so we can merge?  If not I can make the change.

Reply:
I've been working on @JaKXz 's queries about how instrument handles absolute and relative paths in Windows, but life, family and Easter come first.  I think this is the only remaining piece of work for this PR, and my gut feeling says that if Windows can't accept both relative and absolute paths that may be a bug anyway.  So I'll get my Windows test env up and if I find anything that needs to change I'll pop it in a new PR,

Oh and thanks to all of you @JaKXz, @coreyfarrell & @bcoe for your help in getting these instrument changes over the line.  It's been fun.